### PR TITLE
fix test breakages

### PR DESCRIPTION
**Pull request by morgo**
## A Pull Request should be associated with an Issue.

Due to an commit a couple back (introducing a faulty TestMain) the migration package tests were not running. Thankfully only tests broke, no bugs were introduced.

### CODE REVIEW

File: pkg/migration/cutover.go [hunk 1]
```diff
@@ -168,6 +168,7 @@ func (c *CutOver) algorithmGhost(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	defer trx.Rollback() //nolint: errcheck
```
Comment (morgo) [inline pkg/migration/cutover.go:174]: This is a leak I caught.

File: pkg/migration/migration_test.go [hunk 2]
```diff
@@ -15,6 +17,7 @@ import (
 
 func TestMain(m *testing.M) {
 	checkpointDumpInterval = 100 * time.Millisecond
+	os.Exit(m.Run())
```
Comment (morgo) [inline pkg/migration/migration_test.go:21]: This was the fatal mistake. This line was missing.

Comment (prudhvi) [reply pkg/migration/migration_test.go:21]: wow , silent killer, good catch.

File: pkg/migration/cutover.go (hunk 1 above)
Comment (prudhvi) [reply pkg/migration/cutover.go:174]: Shouldn't you rollback only incase of error?

File: pkg/migration/migration_test.go (hunk 2 above)
Comment (prudhvi) [reply pkg/migration/migration_test.go:21]: Should we reset the values back to original after the run() method if we want other package tests to use the original values?

**Review by prudhvi — APPROVED** [review]
LGTM, except have a question about `trx.rollback()`

File: pkg/migration/cutover.go (hunk 1 above)
Comment (morgo) [reply pkg/migration/cutover.go:174]: I will add a comment to make this clearer.

There is no `trx.Close()` (go idiomatic) you have to chose to commit or rollback. This connection is used for `RENAME` which is DDL, and is non-transactional. So either commit or rollback has the same effect. I chose rollback.

File: pkg/migration/migration_test.go (hunk 2 above)
Comment (morgo) [reply pkg/migration/migration_test.go:21]: The other test packages are run concurrently, so they will have to live with these shorter values anyway.

**Pull request merged**